js/scriptIndex.js

```diff
@@ -0,0 +1,16 @@
+window.onresize = function () {
+    console.log(window.outerWidth);
+    if (window.outerWidth < 800) {

```

Comment:
Codacy has a fix for the issue: [Replace `····if·(window.outerWidth·<·800)·{␍` with `··if·(window.outerWidth·<·800)·{`](https://app.codacy.com/gh/Jeffer48/Consultarq/pullRequest?prid=9456241)
```suggestion
  if (window.outerWidth < 800) {
```